cmake/ThirdPartyPackagesKw.cmake

```diff
@@ -20,6 +20,21 @@ if ( LINUX AND KNOWHERE_WITH_DISKANN)
         ${KNOWHERE_THIRDPARTY_DEPENDENCIES}
         DiskANN
         )
+
+    # For Ubuntu 18.04 (install MKL by script)
+    set( CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${MKL_DIR})
+    set(MKL_ROOT "/opt/intel/oneapi/mkl/latest")
+    set(OMP_PATH "/opt/intel/oneapi/compiler/2022.0.2/linux/compiler/lib/intel64_lin/")
+    find_package(MKL REQUIRED)
+
+    # For Ubuntu 20.04/22.04 (install MKL by apt)
+    link_directories(/usr/lib/x86_64-linux-gnu/mkl)
```

Comment:
can the path hard-coded work in different platforms ?

Reply:
ubuntu works, we will replace mkl to openblas in the future.